config/vendor-api-v1.yml

```diff
@@ -497,17 +497,16 @@ components:
       type: object
       additionalProperties: false
       required:
-        - start_date
+        - recruitment_cycle_year
```

Comment:
We'll need to update the release notes too!

Reply:
:+1: Have updated alpha release notes. Only issue with this is that I have to guess when the release will go out (I've guessed tomorrow).

Reply:
Good guess :)